src/main/kotlin/no/nav/familie/ef/sak/behandling/BehandlingRepository.kt

```diff
@@ -18,6 +18,16 @@ interface BehandlingRepository : RepositoryInterface<Behandling, UUID>, InsertUp
 
     fun findByFagsakIdAndStatus(fagsakId: UUID, status: BehandlingStatus): List<Behandling>
 
+    // language=PostgreSQL
+    @Query("""
+        SELECT b.id FROM behandling b 
+            JOIN tilkjent_ytelse ty ON b.id = ty.behandling_id
+            JOIN andel_tilkjent_ytelse aty ON aty.tilkjent_ytelse = ty.id
```

Comment:
Med join med andel_tilkjent_ytelse så kan man risikere å returnere flere rader for samme behandling, som gjør at denne då returnerer en behandlingId flere ganger -> burde kanskje bruke distinct eller bruke set på disse